### DISCUSSION TITLE
Page monitoring on iOS

### DISCUSSION THREAD

**Discussion by hedi94** [lgh06/web-page-monitor#44] [Q&A]
Hello,
I’m using page monitor on google chrome on my computer and it’s working very well.
Unfortunately, I didn’t find any equivalent on iOS. I took chrome app on my iPhone but impossible to install page monitor.
Is there any possibility to do it?
Thank you very much for your great help,
Best regards,
Hedi.

**Comment by lgh06** [comment]
The website, webpagemonitor.net , is just an entry. the real processing procedure is on cloud VPS, not on your local iPhone.

So it will works fine even if you closed your PC / closed browser window.  

So if you are on iPhone, just type webpagemonitor.net in your safari or chrome, for now.
